backend/models/time.py

```diff
@@ -109,18 +109,20 @@ def _measure_one_to_many_points(self, origin, point_list):
         Result is a list of durations in seconds.
         """
         if self.max_matrix_size is None:
-            return self._get_matrix(source_points=[origin], destination_points=point_list)[0]
-
-        measurements = []
-        for batch in iterators.iterate_in_slices(
-            iterable=iter(point_list),
-                batch_size=self.max_matrix_size - 1):
-            measurements.extend(
-                self._get_matrix(
-                    source_points=[origin],
-                    destination_points=batch)[0]
-            )
-        return measurements
+            measurements = self._get_matrix(
+                source_points=[origin],
+                destination_points=point_list)[0]
+        else:
+            measurements = []
+            for batch in iterators.iterate_in_slices(
+                iterable=iter(point_list),
+                    batch_size=self.max_matrix_size - 1):
+                measurements.extend(
+                    self._get_matrix(
+                        source_points=[origin],
+                        destination_points=batch)[0]
+                )
+        return list(filter(None.__ne__, measurements))
```

Comment:
List comp

Reply:
This seems potentially very problematic since the returned list is no longer the same length as the number of destination points. So e.g. using `zip(desintation_points, distances)` will have weird behavior.